index.html

```diff
@@ -1,255 +1,455 @@
-<!doctype html>
 <html lang="en">
-
-<head>
-    <title>Planet CDOT</title>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-    <meta name="generator" content="http://intertwingly.net/code/venus/">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <link rel="stylesheet" href="planet.css" type="text/css">
-    <!-- <link rel="alternate" href="http://zenit.senecac.on.ca/~chris.tyler/planet/atom.xml" title=""
-            type="application/atom+xml"> -->
-    <!-- CSS Bootstrap 4.0 -->
-
-    <!-- Font Awesome -->
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
-    <!-- Bootstrap core CSS -->
-    <link href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet">
-    <!-- Material Design Bootstrap -->
-    <link href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.8.10/css/mdb.min.css" rel="stylesheet">
-
-
-</head>
```

Comment:
This doesn't needed to be removed. I see that you added the exact same lines after so it won't effect the results but this code was originally added by two other contributors. Let's not alter the git history.

Reply:
I did not remove or move those lines (4 to 22) that you are referring to. They are still present. Only their alignment has been corrected when I made my own additions to the file as per standard HTML conventions.  As you said, this has no effect on the actual output, nor has it changed any of the work of the original contributors.
![image](https://user-images.githubusercontent.com/38863472/68634093-4ec63300-04c2-11ea-9fca-dad7ef3eb53f.png)